Remotion/Web/Development.WebTesting/WebDriver/Factories/Chrome/ChromeBrowserFactory.cs

```diff
@@ -55,7 +55,7 @@ public IBrowserSession CreateBrowser (DriverConfiguration configuration)
 
       var extendedChromeOptions = _chromeConfiguration.CreateChromeOptions();
       if (configuration.Headless)
-        extendedChromeOptions.AddArgument("headless");
+        extendedChromeOptions.AddArgument("headless=new");
```

Comment:
That's just...yeah, okay, Chromium. Whatever :p

Great docs in the Jira ticket, btw!